src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -349,6 +349,47 @@ export default function ({
     });
   }
 
+  function getConditionalTemplate(term: { value?: string; context?: AutoCompleteContext }) {
+    const context = term.context;
+    if (context?.endpoint && term.value) {
+      const { data_autocomplete_rules: autocompleteRules } = context.endpoint;
+      if (autocompleteRules && autocompleteRules[term.value]) {
+        const currentLineNumber = editor.getCurrentPosition().lineNumber;
+        const rules = (autocompleteRules[term.value] as any).__one_of ?? [];
```

Comment:
Again, why `any` is here? If some type is missing, you should add it. Otherwise, you should "proof", that the value is of some time. It is unacceptable to use `any` in the code.